packages/react/src/dialog-content/DialogContent.css.ts

```diff
@@ -22,14 +19,27 @@ export const content = recipe({
   variants: {
     size: {
       sm: style({
+        borderRadius: "8px",
+        maxHeight: "75dvh",
+        maxWidth: "calc(100dvw - 2 * 24px)",
```

Comment:
let's move this back to the base style instead of repeating it

and reset the styles in the fullscreen branch - like use `0` for radius and `none` for max-height/max-width